documentation/using/master.adoc

```diff
@@ -7,8 +7,6 @@ include::shared/attributes.adoc[]
 
 include::assemblies/assembly-overview.adoc[leveloffset=+1]
```

Comment:
At the beginning of the _Using Strimzi_ guide, I suggest adding a note that signposts users to the ex-getting started content. Something similar to this might work:

NOTE: Looking to get started with Strimzi? For step-by-step deployment instructions, see the _Deploying Strimzi_ guide.

Reply:
Nice idea!